vagrant/Vagrantfile

```diff
@@ -0,0 +1,11 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure(2) do |config|
+  config.vm.box = "debian/contrib-jessie64"
```

Comment:
Generally I follow the rule of thumb "Debian on the server, Ubuntu on the desktop" but if devs prefer Ubuntu I don't mind. The only concern is systemd: if it's already available on Ubuntu 14.04 then all should work out. Gotta try, if it's there then should make no difference, otherwise it will make the process start/stop/restart management more complex